cmd/import.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e;
+
+# per-source imports
+function import_wof(){ compose_run 'whosonfirst' npm start; }
+function import_oa(){ compose_run 'openaddresses' npm start; }
+function import_osm(){ compose_run 'openstreetmap' npm start; }
+function import_polylines(){ compose_run 'polylines' npm start; }
+function import_transit(){ compose_run 'transit' npm start; }
+
+register 'import' 'wof' '(re)import whosonfirst data' import_wof
+register 'import' 'oa' '(re)import openaddresses data' import_oa
+register 'import' 'osm' '(re)import openstreetmap data' import_osm
+register 'import' 'polylines' '(re)import polylines data' import_polylines
+register 'import' 'transit' '(re)import transit data' import_transit
+
+# import all the data to be used by imports
+# note: running these in parallel requires a large amount of RAM (so we dont)
```

Comment:
The CPU can be an issue as well, as Elasticsearch might not be able to handle all the incoming requests. Maybe we rewrite this line with something more generic like "running importers in parallel can cause issues"